Initial Check for TDD with JIRA ticket: TP-27

fahad-ahmed123 opened the pull request:
Code review for League Service TDD

kbolishetti committed:
Initial Check for TDD with JIRA ticket: TP-27

payweltechnut reviewed (changes requested):
PP TDD session comments need to be addressed

kbolishetti committed:
Checkin after Comment fixes suggested by Pawel. 

*Fixed the format of exception tests.
*Added Test cases for Validate Method
*Added Test cases for JoinLeague Method

fahad-ahmed123 commented:
@payweltechnut 
Hi Pawel,  
I have updated the test cases based on your comments. Please review and provide your feedback.  
Thanks !

kbolishetti committed:
Initial Commit for Workspace Fix for TP-27.
